charts/linkurious-enterprise/templates/deployment.yaml

```diff
@@ -150,7 +150,7 @@ spec:
         - name: litestream
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: litestream/litestream:0.3.9
+          image: {{ .Values.backup.litestream.image }}
```

Comment:
I previously updated the init container and didn't see there was two reference of litestream image...

Reply:
Haha it didn't tick either